docs/_source/docs/cli.rst

```diff
@@ -6,64 +6,11 @@ format:
 
 .. code:: shell
 
-    $ sceptre [GLOBAL_OPTIONS] COMMAND [ARGS] [COMMAND_OPTIONS]
+    $ sceptre
```

Comment:
Can you please revert this line back to include `sceptre [GLOBAL_OPTIONS] COMMAND [ARGS] [COMMAND_OPTIONS]`? Was there a reason you just kept `$sceptre` here?